packages/pn-personafisica-webapp/src/redux/notification/__test__/reducers.test.ts

```diff
@@ -202,4 +206,34 @@ describe('Notification detail redux state tests', () => {
     expect(action.type).toBe('getNotificationPaymentUrl/fulfilled');
     expect(payload).toEqual({ checkoutUrl: 'mocked-url' });
   });
+
```

Comment:
it would be nice to have such tests also for `pn-personagiuridica-webapp`

Reply:
done